src/pages/Explore/Explore.module.css

```diff
@@ -0,0 +1,90 @@
+.videos_container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(14rem, 1fr));
+  /* flex-wrap: wrap;
+  justify-content: flex-start; */
```

Comment:
You can remove comments if it's no longer needed.

Reply:
Will do that